src/Common/src/System/IO/StringBuilderCache.cs

```diff
@@ -66,6 +66,13 @@ public static StringBuilder Acquire(int capacity = DEFAULT_CAPACITY)
             return new StringBuilder(capacity);
         }
 
+        public static StringBuilder Acquire(string value)
```

Comment:
Without extension statics or extension constructors, it'd be awkward to do so for the static Acquire, which is essentially a factory.  And I think it's good for readability/understandability for Release/GetStringAndRelease to be called in the same manner that Acquire is called, e.g. I think it'd be confusing to see code like:
```C#
    StringBuilder sb = StringBuilderCache.Acquire();
    /// ... some appends
    sb.Release();
```
so I'd be inclined to leave these APIs as they are, at least for now.